src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -10,6 +10,24 @@ const createZomeCall = instanceCreateZomeCall(INSTANCE_ID)
 
 const MOCK_DEADLINE = '4019-01-02T03:04:05.678901234+00:00'
 
+// Creates an array of all counterparties for a provided transaction list
+export async function getTxCounterparties (transactionList) {
```

Comment:
I'm not seeing where this function is used. Is this a left over? Or am I missing something?

Reply:
100% left over from the 'array' approach to resolving this useQuery (with param) call within our current component patterns.